### ISSUE TITLE
Calls to `/api/v1/status` result in error 500

### ISSUE THREAD

**Issue by peeech** [Holo-Host/holo-nixpkgs#367]
Calls to `/api/v1/status` result in error 500

**to reproduce**
- checkout #365 and build HPOS
- register HPOS
- access HP-Admin UI, login and navigate to `/admin/settings`
- observe the log

Client side error:
```
https://....holohost.net/api/v1/status
Status Code: 500 Internal Server Error
```
Server side error:
```
[...] hpos hpos-admin[969]:  FileNotFoundError: [Errno 2] No such file or directory: 'sudo'
```

**my theory**
[this line of code](https://github.com/Holo-Host/holo-nixpkgs/blob/2020-02-03-rc-1/overlays/holo-nixpkgs/hpos-admin/hpos-admin.py#L66) needs some tweaks

**Comment by peeech** [comment]
cc: @yegortimoshenko

**Comment by peeech** [comment]
Resolved in #368
